styles.css

```diff
@@ -489,10 +543,317 @@ hr {
+  }
+
+  .nav-bar {
+    width: 100%;
+  }
+
+  .nav-header {
+    align-items: flex-end;
+  }
+
+  .nav-list {
+    flex-direction: row;
+    margin: 0 150px 0 600px;
+  }
+
+  .nav-items {
+    margin: 0 10px;
+  }
+
+  .items {
+    width: 46px;
+    height: 20px;
+    color: #344563;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .items:hover {
+    color: #6070ff;
+  }
+
+  .logo {
+    width: 76px;
+    height: 20px;
+    margin: 10px 0 0 155px;
+  }
+
+  #menu-button {
+    display: none;
+  }
+
+  .hidden {
+    display: flex;
+  }
+
+  .main-container {
+    margin: 1px 1px;
+    background: #fff;
+    border-radius: 0 0 0 150px;
+  }
+
+  .main {
+    background: url('./assets/Header-bg.png');
+    background-repeat: no-repeat;
+    background-size: 100%;
+    width: 90%;
+    margin: 10px 0 0 15px;
+    height: 650px;
+  }
+
+  .text-section {
+    text-align: left;
+    width: 720px;
+    height: 268px;
+    margin: 50px 130px;
+  }
+
+  .main-header {
+    flex-direction: row;
+    font-size: 48px;
+    line-height: 60px;
+    height: 122px;
+  }
+
+  .main-text {
+    flex-direction: row;
+    font-size: 20px;
+    height: 98px;
+  }
+
+  .contact {
+    width: 130px;
+    height: 24px;
+    margin: 12px 0;
+  }
+
+  .icons {
+    width: 20px;
+    height: 20px;
+  }
+
+  /* card sections */
+  .card {
+    flex-direction: row;
+    height: 496px;
+    width: 1150px;
+    padding: 24px;
+    margin: 142px 5px 5px 30px;
+  }
+
+  .card-items {
+    flex-direction: row;
+  }
+
+  .card-items-reverse {
+    flex-direction: row-reverse;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    margin: 12px 0;
+  }
+
+  .card-continent {
+    width: 540px;
+    height: 415px;
+    margin: 16px 22px;
+    padding: 12px;
+    right: 0;
+  }
+
+  .card-image {
+    width: 540px;
+  }
+
+  .card-header {
+    width: 400px;
+    font-size: 40px;
+    line-height: 52px;
+    margin: 20px 0;
+  }
+
+  .card-text {
+    width: 496px;
+    height: 50px;
+    margin: 25px 0;
+  }
+
+  .skils {
+    flex-direction: row;
+    width: auto;
+    height: 16px;
+    text-align: center;
+  }
+
+  #special-button {
+    display: flex;
+    color: #fff;
+    background: #4053fc;
+    box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
+    border-radius: 8px;
+  }
+
+  #special-button:hover {
+    color: #4053fc;
+    background: #fff;
+    border: 0.5px solid #4053fc;
+  }
+
+  .button-container {
+    margin: 0 0 0 -10px;
+  }
+
+  .button {
+    width: auto;
+  }
+
+  .cursor {
+    width: 57px;
+    height: 57px;
+    margin: 50px 0 0 320px;
+  }
+
+  /* About section */
+  .about {
+    flex-direction: row;
+    width: 100%;
+    height: 924px;
+    border-radius: 0 150px 0 0;
+    margin: 142px 130px 142px -10px;
+  }
+
+  .about-items {
+    flex-direction: row;
+    height: 496px;
+    margin: 50px 0 0 50px;
+  }
+
+  .about-continent {
+    width: 447px;
+  }
+
+  .about-text {
+    height: 102px;
+    width: 447px;
+  }
```

Comment:
- [ ] Kindly replicate the design as much as possible (e.g., font, colors, images, tests, margins)
Your design
![Capture1](https://user-images.githubusercontent.com/85095702/159911727-bc5d73ca-3db9-45e0-8790-5078f2939233.PNG)

Figma template
![figma3](https://user-images.githubusercontent.com/85095702/159911790-f326e6c6-013b-4a04-bef1-131bf5bcc56a.PNG)